M2/assignment3_m1.py

```diff
@@ -93,6 +100,32 @@ def tokenize_content(content):
         stemmed_tokens.append(ps.stem(token.lower()))
     return stemmed_tokens
 
+def tokenize_content_without_stopwords(content):
+    stop_words = set(stopwords.words('english'))
+    ps = PorterStemmer()
+    content = re.sub(r"[^a-zA-Z0-9\n]", " ", content)
+    tokens = word_tokenize(content)
+    stemmed_tokens = []
+    for token in tokens:
+        if len(token) < 3 or token in stop_words :
+            continue
+        stemmed_tokens.append(ps.stem(token.lower()))
+    return stemmed_tokens
+
+def generate_bigram_tokens(tokens):
+    bigram_tokens = []
+    bigrams_list = list(nltk.bigrams(tokens))
```

Comment:
Pretty clever way of generating n-gram tokens. I was curious how you will do that.Nice!